appveyor.yml

```diff
@@ -9,15 +9,16 @@ assembly_info:
   assembly_version: '{version}.0'
```

Comment:
Change still needed here (without it we'll use varying .NET Core versions depending on build machine config, IIRC).